src/modules/verifiableCredentials/verifiableCredentials.types.ts

```diff
@@ -48,3 +49,36 @@ export interface VerifyVerifiableCredentialResults {
   warnings: string[];
   errors: string[];
 }
+
+// Verifiable Presentation Types
+
+export interface SignVerifiablePresentationOptions {
+  verificationMethod?: string;
+  proofPurpose?: string;
+  domain?: Partial<TypedDataDomain>;
+}
+
+export interface SignedVerifiablePresentation {
```

Comment:
`Signed` is a bit redundant as VerifiablePresentation needs to have a `proof`
```suggestion
export interface VerifiablePresentation {
```